accounts/tests.py

```diff
@@ -1,3 +1,29 @@
-# from django.test import TestCase
+import pytest
+from django.contrib.auth.models import User
+from .models import UserAccount
 
-# Create your tests here.
+
+@pytest.fixture
+def test_account(db):
+    """test account to reduce redundancy"""
+    username = 'testuser'
+    password = 'lvkmdgv32fdbf1b'
+    email = 'testuser@test.com'
+    new_user = User.objects.create_user(
+        username=username, password=password, email=email
+    )
+    return new_user
+
+
+@pytest.mark.django_db()
+def test_user_create(test_account):
+    """Create a user and check if a UserAccount instance was created"""
+    assert User.objects.count() == 1
+    assert UserAccount.objects.count() == 1
+
+
+@pytest.mark.django_db()
+def test_user_find(test_account):
+    """Find user in database by user id, username or email"""
+    assert User.objects.get(username=test_account.username)
+    assert User.objects.get(email=test_account.email)
```

Comment:
if you found that user by username there is no reason you wouldn't find it by email, so the second assert seems redundant. @benipeled @bardielle thoughts?